src/controllers/controllermappinginfo.h

```diff
@@ -13,15 +13,13 @@ struct ProductInfo {
     QString protocol;
     QString vendor_id;
     QString product_id;
-
-    // HID-specific
-    QString in_epaddr;
-    QString out_epaddr;
-
-    // Bulk-specific
     QString usage_page;
     QString usage;
     QString interface_number;
+
+    // Bulk-specific
+    QString in_epaddr;
+    QString out_epaddr;
```

Comment:
```suggestion
    QString interface_number;
    
    // HID-specific
    QString usage_page;
    QString usage;

    // Bulk-specific
    QString in_epaddr;
    QString out_epaddr;
```
Every USB device has interface numbers, but only HID works with Uage/UsagePage